war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1382,7 +1382,7 @@ function updateOptionalBlock(c,scroll) {
     while (!vg.hasClassName("rowvg-start"))
         vg = vg.next();
 
-    var checked = xor(c.checked,Element.hasClassName(c,"negative"));
+    var checked = xor(c.checked,Element.hasClassName(c,"negative")) || Element.hasClassName(c, "display-only-checked");
```

Comment:
Could `Element.hasClassName(c,"negative")) || Element.hasClassName(c, "display-only-checked"` be extracted into a variable? I think this conditional complicates the `xor` function a bit.

Reply:
this isn't actually in use anymore, I need to revert it.

I was trying to not use the native controls and use icon's / images so that users don't get confused due to disabled controls not being that obvious.

But the javascript wasn't the easiest to change and I didn't want to break anything so I went back to using the native controls but forgot to revert this change